### PR TITLE
Feat/hide deprecated fe functions

### PR DESCRIPTION
**Pull request by adamjezek98**

### CODE REVIEW

File: src/reducers/gcodes.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default (
         pages = [newPage];
       }
       // This line does everything so far:
-      pages = {data: {items: action.payload.data}, startWith: null};
+      pages = { data: { items: action.payload.data }, startWith: null };
```
Comment (czervenka) [inline src/reducers/gcodes.js:51]: The spaces inside object literal are the version preferred by linter?

**Review by czervenka — APPROVED** [review]
Looks good to me although I'm not used to read React. @pavel-mxsf could you double-check, please?

File: src/services/backend/gcodes.js [hunk 2]
```diff
@@ -66,11 +68,11 @@ export const uploadGcode = (orgId, path, file) => {
   var data = new FormData();
```
Comment (czervenka) [inline src/services/backend/gcodes.js:68]: `path` argument is not used anymore and should be removed, right?

**Review by czervenka — CHANGES_REQUESTED** [review]

**Review by pavel-mxsf — APPROVED** [review]

File: src/reducers/gcodes.js (hunk 1 above)
Comment (pavel-mxsf) [reply src/reducers/gcodes.js:51]: Yes, it's ok.

File: src/components/listings/wrapper.js [hunk 3]
```diff
@@ -118,24 +124,25 @@ class Wrapper extends React.Component {
...
-                  itemList.orderBy === `+${column}`
-                    ? `-${column}`
-                    : `+${column}`,
-                  itemList.filter,
-                  itemList.limit
-                );
-              };
-            }}
-          />
+          {enableSorting !== false && (
```
Comment (pavel-mxsf) [inline src/components/listings/wrapper.js:127]:
```suggestion
          {enableSorting && (
```

**Pull request merged**